cmd/client-fs.go

```diff
@@ -670,36 +632,8 @@ func (f *fsClient) listPrefixes(prefix string, contentCh chan<- *clientContent)
 		if fi.Mode()&os.ModeSymlink == os.ModeSymlink {
 			st, e := os.Stat(file)
 			if e != nil {
-				if os.IsPermission(e) {
-					contentCh <- &clientContent{
-						Err: probe.NewError(PathInsufficientPermission{
-							Path: pathURL.Path,
-						}),
-					}
-					continue
-				}
-				if os.IsNotExist(e) {
-					contentCh <- &clientContent{
-						Err: probe.NewError(BrokenSymlink{
-							Path: pathURL.Path,
-						}),
-					}
-					continue
-				}
-				if strings.Contains(e.Error(), "too many levels of symbolic links") {
-					contentCh <- &clientContent{
-						Err: probe.NewError(TooManyLevelsSymlink{
-							Path: pathURL.Path,
-						}),
-					}
-					continue
-				}
-				if e != nil {
-					contentCh <- &clientContent{
-						Err: probe.NewError(e),
-					}
-					continue
-				}
+				// Ignore any errors on symlink
+				continue
```

Comment:
And also passing `clientContent.Err` will help in logging specific errors in `doList` - https://github.com/minio/mc/blob/master/cmd/ls.go#L117. This change will not log these errrors. is that ok?

Reply:
Yes it is okay, we don't need to log errors unless users can do something about it. Right now we happily ignore it anyways so we are wasting a call on os.Lstat() which is not meaningful. 

I say we ignore all broken symlinks because those are useless for us.